integration-tests/tests/api/artifacts-cdn.spec.ts

```diff
@@ -313,7 +316,7 @@ function runArtifactsCDNTests(
         `artifact/${target.id}/services`,
       );
       expect(artifactContents.body).toMatchInlineSnapshot(
-        `"[{"name":"ping","sdl":"type Query { ping: String }","url":"ping.com"}]"`,
+        '"[{\\"name\\":\\"ping\\",\\"sdl\\":\\"type Query { ping: String }\\",\\"url\\":\\"ping.com\\"}]"',
```

Comment:
The former was easier to read, do we need to use single quotes?

Reply:
It was auto update snapshots, I can add serializer to remove redundant escapes like in graphql-eslint https://github.com/B2o5T/graphql-eslint/blob/cfbb649f89b76673b8fdd718d3c227bada19a4a6/serializer.js